### PR TITLE
Feature/sig29 filter

### PR DESCRIPTION
**Pull request by aditudorache**
## SIA

Before opening a pull request, please ensure:

- [ ] double-check your branch is based on `develop` and targets `develop` 
- [ ] Pull request has tests (we are going for 100% coverage!)
- [ ] Code is well-commented, linted and follows project conventions
- [ ] Internal code generators and templates are updated (if necessary)

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: src/signals/incident-management/components/FieldControlWrapper/index.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+//   hasError: PropTypes.func.isRequired,
+//   // touched: PropTypes.boolean,
+//   // control: PropTypes.shape({
+//   //   pristine: PropTypes.boolean,
+//   // }),
+//   placeholder: PropTypes.string
+// };
+
+// export default withFieldControl(TextInputComponent);
+
```
Comment (jasperswart) [inline src/signals/incident-management/components/FieldControlWrapper/index.js]: deze kan weg toch?

File: src/signals/incident-management/components/TextInput/index.js [hunk 2]
```diff
@@ -2,8 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import { FieldControl } from 'react-reactive-form';
+// import withFieldControl from '../FieldControlWrapper';
```
Comment (jasperswart) [inline src/signals/incident-management/components/TextInput/index.js]: weg

File: src/signals/incident-management/containers/IncidentCategoryContainer/index.test.js [hunk 3]
```diff
@@ -1,10 +1,38 @@
-// import React from 'react';
-// import { shallow } from 'enzyme';
+import React from 'react';
+import { shallow } from 'enzyme';
 
-// import { IncidentCategoryContainer } from 'index';
```
Comment (jasperswart) [inline src/signals/incident-management/containers/IncidentCategoryContainer/index.test.js]: weg

File: src/signals/incident-management/components/FieldControlWrapper/index.js (hunk 1 above)
Comment (aditudorache) [reply src/signals/incident-management/components/FieldControlWrapper/index.js]: not yet, will be finished later

File: src/signals/incident-management/components/TextInput/index.js (hunk 2 above)
Comment (aditudorache) [reply src/signals/incident-management/components/TextInput/index.js]: not yet, will be finished later

File: src/signals/incident-management/containers/IncidentCategoryContainer/index.test.js (hunk 3 above)
Comment (aditudorache) [reply src/signals/incident-management/containers/IncidentCategoryContainer/index.test.js]: not yet, will be finished later

**Pull request merged**